CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # CHANGE LOG
 
-## 3.15.0-beta.2
```

Comment:
Since we're cleaning up beta notes would you mind removing the 3.14 beta notes as well?